README.md

```diff
@@ -48,6 +48,7 @@
 | 5.      | Checkers with AI      | https://www.youtube.com/playlist?list=PLzMcBGfZo4-myY28wdQuJDBi8pCt-GIj6 | <img src="https://img.shields.io/badge/-medium-orange">  |
 | 6.      | Hangman               | https://www.youtube.com/playlist?list=PLzMcBGfZo4-ndZlN21DasvpfKwIc1rI6w | <img src="https://img.shields.io/badge/-medium-orange">  |
 | 7.      | Flappy Bird with AI   | https://www.youtube.com/playlist?list=PLzMcBGfZo4-lwGZWXz5Qgta_YNX3_vLS2 | <img src="https://img.shields.io/badge/-medium-orange">  |
+| 8.      | Space invaders [Medium] | https://www.youtube.com/watch?v=FfWpgLFMI7w | <img src="https://img.shields.io/badge/-medium-orange">  |
```

Comment:
Instead of **[Medium]** write (Free Code Camp). It would be easy to differentiate.

Reply:
Alright

Reply:
I have fixed the issue .